### ISSUE TITLE
Need help testing Linux & Mac builds

### ISSUE THREAD

**Issue by BBC-Esq** [BBC-Esq/ChromaDB-Plugin-for-LM-Studio#11]
I need some help testing scripts from time to time on both Linux systems with an Nvidia or AMD GPU, and on MacOS systems, using Metal/MPS and not.

Also, would love to be the first to include support for Intel Arc gpu-acceleration as well.

If anyone is interested please respond to this "issue" or e-mail me directly per my e-mail on the readme.

**Comment by abcnow** [comment]
I have a mac.. and I would love to help you test.. just tell me what you need me to do

Apple M2 Max
96 gb ram
with gpu

**Comment by abcnow** [comment]
ps: tried to install this on my mac.. and I get this error:
---
```
python3 gui.py
Traceback (most recent call last):
  File "/Users/ac/Documents/mc2/software/databases/chroma/chroma_for_llmstudio/ChromaDB-Plugin-for-LM-Studio-main/src/gui.py", line 1, in <module>
    from PySide6.QtWidgets import (
ModuleNotFoundError: No module named 'PySide6'
(/Users/ac/Documents/mc2/software/databases/chroma/chroma_for_llmstudio/ChromaDB-Plugin-for-LM-Studio-main/src/conda_chroma_env) 
```

**Comment by abcnow** [comment]
it may be that I am doing something wrong.. but if you want I can give you access to the computer and then you can test it yourself and I get to see how to do it for future events...

**Comment by BBC-Esq** [comment]
First of all, thanks for being willing to test.  If you're interested in testing versions that haven't been formally released yet we can possibly do that too.  I'll look into the error message you encountered, but can you create a new issue using the bug label so we can organize our conversation there...or I'm on the LM Studio discord as well.

**Comment by RuiPedroLeal** [comment]
Hello, I have a M1 Mac and I'm trying to run this. Installation was perfectly ok, but when I execute the GUI ("python3 gui.py" ) I got the following error: 

![image](https://github.com/BBC-Esq/ChromaDB-Plugin-for-LM-Studio/assets/1958613/ea8b39e4-69d8-4bbf-b310-14419677cfba)

Can I disable the CUDA support on the translate package to just use CPU?

Great work, btw!

**Comment by BBC-Esq** [comment]
Should be fixed as of version 2.7.

**Comment by BBC-Esq** [comment]
Closing for lack of interest.
